internal/transform/account.go

```diff
@@ -21,6 +21,11 @@ func TransformAccount(ledgerChange ingest.Change) (AccountOutput, error) {
 		return AccountOutput{}, fmt.Errorf("Could not extract account data from ledger entry; actual type is %s", ledgerEntry.Data.Type)
 	}
 
+	outputCloseTime, err := utils.TimePointToUTCTimeStamp(ledgerCloseMeta.MustV0().LedgerHeader.Header.ScpValue.CloseTime)
```

Comment:
We need this to support both Meta types: `V0` and `V1`. Can you refactor to also return the `CloseTime` for `MustV1()`?

Reply:
I would also consider making this a method in `utils` so you can just pass LedgerCloseMeta from each transform to return the closedAt time.